internal/node_install.bzl

```diff
@@ -58,26 +60,41 @@ def _node_impl(repository_ctx):
   executables = get_executables(repository_ctx)
   repository_ctx.file("BUILD.bazel", content="""#Generated by node_install.bzl
 package(default_visibility = ["//visibility:public"])
-exports_files([
-  "{0}",
-  "{1}",
-])
+exports_files(["{0}"])
 alias(name = "node", actual = "{0}")
-alias(name = "npm", actual = "{1}")
-""".format(executables["node"], executables["npm"]))
+sh_binary(
```

Comment:
we have to avoid bash dependencies - but since we're underneath nodejs binary here I think that means authoring this again in Windows Batch
don't need to fix in this PR